README.md

```diff
@@ -60,4 +68,49 @@ Get in touch today! [Book a demo](https://xygeni.io/book-a-demo?utm_source=jenki
     * Set the `Xygeni API Url` (https://api.xygeni.io).
     * Optionally click the `TestToken` button to check url connection and token validity.
     * Click the `Save` button. 
-    ![Xygeni Configuration](docs/images/xygeni-config.png)
\ No newline at end of file
+    ![Xygeni Configuration](docs/images/xygeni-config.png)
+
+
+
+# Xygeni Sensor Unusual Activity
+
+By installing the Xygeni sensor, user activities and actions will be monitored and suspicious events will be sent to the Xygeni server for analysis.
+
+
+# XygeniSalt Step
+
+## Use Xygeni Salt Step to generate Attestation Provenance
+
+The Xygeni Sensor plugin add a Build Step to generate [SLSA provenance attestations](https://slsa.dev/provenance/).
+This step command will generate an slsa attestation provenance ``xygeni-salt-attestation.json`` in SLSA format and upload attestation to ``salt.xygeni.io`` server.
+
+## Job configuration - Freestyle project
+
+The plugin provides a ```Post-build action``` which will generate SLSA provenace attestations.
+
+**Artifact Path Filter**: Specifies the artifacts to include.
+
+![xygenisalt-provenance.png](docs/images/xygenisalt-provenance.png)
+
+**Manually defined Subject Attestation**: Specifies one or more name and content to include (a string, file or image).
+
+![xygenisalt-manual.png](docs/images/xygenisalt-manual.png)
+
+**Signer configuration**: Add reference or enviroment variable to configure the key, public key and password. Optionally a certificate could be added to avoid a round to verification.
+
+![xygenisalt-signer.png](docs/images/xygenisalt-signer.png)
+
+## Job configuration - Pipeline project
+
```

Comment:
Documentation to add for the xygeniSalt command:

---
The `xygeniSalt` step could be invoked for generating SALT provenance. Build information for the registered attestation **subjects** (also known as software 'products' or 'artifacts')  will be registered in the signed attestation.

Subjects could be provided explicitly as a list in the `subjects` property. Each item in the list is a map with a given `name`and a content, either a Docker image published in a remote registry as part of the build (`image: 'REGISTRY/IMAGE_NAME:TAG'`), a local file produced by the build as a packaged artifact (`file: 'path/to/file.zip'`), or a value (which could be a SHA digest of a given artifact, a base-64 encoded binary value, or a string representing the artifact (`value: 'sha:03afb3...1c24'`).

Alternatively, subjects could be referenced by pattern using the `artifactFilter` pattern (an Ant-like pattern), which matches files in the workspace that will be used as subjects for the SLSA provenance attestation.  

The `key` / `publicKey` parameters contain the key pair to use for signing the provenance file. The signature is done with the private key, and the public key is added to the signed attestation for verification by software consumers. The keys are provided as either PEM-encoded values (text format with key enclosed between -----BEGIN...----- and -----END ...----- delimiters). The PEM-encoded key could be provided also as a path to the key file relative to the workspace directory (prefix the path with `file:` prefix), or as an environment variable prefixed with `env:`.

An optional X.509 certificate could be used for helping the software consumer to trust the signature, using the `certificate` parameter with a similar format.

The `pkiFormat` specifies the signature format (one of `x509`, `minisign`, `ssh`, `pkcs7` or `tuf`). Use `x509` as a good default.

Reply:
added